taskcluster/docker/bump/owner_trust.db

```diff
@@ -0,0 +1,3 @@
+# List of assigned trustvalues
+# (Use "gpg --import-ownertrust" to restore them)
+14F26682D0916CDD81E37B6D61B7B526D98F0353:6:
```

Comment:
We ultimately trust (aka `6`) this key because `gpg --import-ownertrust` doesn't understand any lower value. Thus, `gpg --verify` errors out because the key doesn't have enough trust, even if imported. I don't see this as a big deal because it's the only imported key in the keyring.